web/project.jsp

```diff
@@ -825,116 +831,85 @@
-                                        }
-                                        if (toolName[18] == null) {
-                                            toolName[18] = "Enigma";
-                                        }
-                                        if (toolName[19] == null) {
-                                            toolName[19] = "Latin Dictionary";
                                         }
-                                        out.print(toolOptions.toString());
                                     %>
-                                    <label class='projectTools'><input name="projectTool[]" type="checkbox" <%out.print(toolCheck[20]);%> value="https://centerfordigitalhumanities.github.io/cappelli/" title="Cappelli's Abbreviations"/><span contentEditable="true"><%out.print(toolName[20]);%></span></label>
-                                    <label class='projectTools'><input name="projectTool[]" type="checkbox" <%out.print(toolCheck[14]);%> value="http://vulsearch.sourceforge.net/cgi-bin/vulsearch" title="Latin Vulgate Search"/><span contentEditable="true"><%out.print(toolName[14]);%></span></label>
-                                    <label class='projectTools'><input name="projectTool[]" type="checkbox" <%out.print(toolCheck[19]);%> value="http://www.perseus.tufts.edu/hopper/resolveform?lang=latin" title="Latin Dictionary"/><span contentEditable="true"><%out.print(toolName[19]);%></span></label>
-                                    <label class='projectTools'><input name="projectTool[]" type="checkbox" <%out.print(toolCheck[15]);%> value="http://quod.lib.umich.edu/m/med/lookup.html" title="Middle English Dictionary"/><span contentEditable="true"><%out.print(toolName[15]);%></span></label>
-                                    <label class='projectTools'><input name="projectTool[]" type="checkbox" <%out.print(toolCheck[16]);%> value="http://www.cnrtl.fr/morphologie/" title="French Dictionary"/><span contentEditable="true"><%out.print(toolName[16]);%></span></label>
-                                    <label class='projectTools'><input name="projectTool[]" type="checkbox" <%out.print(toolCheck[17]);%> value="http://tapor.library.utoronto.ca/cgi-bin/doe/production.pl" title="Dictionary of Old English"/><span contentEditable="true"><%out.print(toolName[17]);%></span></label>
-                                    <label class='projectTools'><input name="projectTool[]" type="checkbox" <%out.print(toolCheck[18]);%> value="http://ciham-digital.huma-num.fr/enigma/" title="Enigma"/><span contentEditable="true"><%out.print(toolName[18]);%></span></label>
+                                    <label class='projectTools'><input name="projectTool[]" type="checkbox" <%out.print(projectToolCheck[0]);%> value="https://centerfordigitalhumanities.github.io/cappelli/" title="Cappelli's Abbreviations"/><span contentEditable="true">Cappelli's Abbreviations</span></label>
```

Comment:
This is okay, but it causes unexpected things to happen if a tool is inserted into the mix later. The label and link were connected for a good reason.